content/docs/command-reference/remote/modify.md

```diff
@@ -758,6 +758,55 @@ more information.
 
 </details>
 
+<details>
+
+### Click for OSF
+
+- `url` - remote location:
+
+  ```dvc
+  $ dvc remote modify osfremote url osf://osf.io/path/to/dir
+  ```
+
+- `project` - [OSF](https://osf.io/) project name. If you open a page
+   with your project in your browser, you can see it in the
+   address bar : `https://osf.io/<your_project>/`.
+
+  ```dvc
+  $ dvc remote modify osfremote --local project <project_name>
+  ```
+  Also it is possible to specify `project` by setting
+  envirement variable `OSF_PROJECT`. Run
+  ```$ export OSF_PROJECT=<your_project>```
+  
+- `user` - OSF account username (email address by default):
+
+  ```dvc
+  $ dvc remote modify osfremote --local user example@mail.com
+  ```
+  Also it is possible to specify `user` by setting
+  envirement variable `OSF_USER`. Run
+  ```$ export OSF_PROJECT=example@mail.com```
```

Comment:
More spacing and open code block, and a typo fix:
````suggestion

  Also it is possible to specify `project` by setting
  environment variable `OSF_PROJECT`. Run
  `export OSF_PROJECT=<your_project>`

- `user` - OSF account username (email address by default):

  ```dvc
  $ dvc remote modify osfremote --local user example@mail.com
  ```

  Also it is possible to specify `user` by setting
  environment variable `OSF_USER`. Run
  `export OSF_PROJECT=example@mail.com`
````